http-clients/aws-crt-client/src/it/java/software/amazon/awssdk/http/crt/AwsCrtClientKmsIntegrationTest.java

```diff
@@ -43,9 +47,13 @@ public void setup() {
                 continue;
             }
 
+            int numThreads = 1;
+            eventLoopGroup = new EventLoopGroup(numThreads);
```

Comment:
I"m not sure what the purposes of these changes is. Care to enlighten me?

Reply:
The client doesn't create its own event loop group anymore, as people felt it was a bit awkward for them to create such a heavy object.  Because of that, we don't have the .eventLoopSize(1) builder change that we used to have on the client, and this is meant to be the replacement of that.  (The numThreads variable is just meant to be more explicit about what's being passed in, and could arguably be taken out or made final.)